FirebaseVertexAI/Sources/FunctionCalling.swift

```diff
@@ -25,7 +25,7 @@ public struct FunctionDeclaration {
   /// A brief description of the function.
   let description: String
 
-  /// Describes the parameters to this function; must be of type ``DataType/object``.
+  /// Describes the parameters to this function; must be of type `"OBJECT"`.
```

Comment:
what's OBJECT?

Reply:
Since this is an internal property I could change this to `DataType.object`. The type `DataType` is also internal now so I can't use the ``'s to reference it anymore. This is what it's referring to:
https://github.com/firebase/firebase-ios-sdk/blob/6059bbbc98a488c096d17c002fae181ab6a548e1/FirebaseVertexAI/Sources/Types/Internal/DataType.swift#L34-L35

I had changed it to `"OBJECT"` since we're now exposing the public `Schema.type` as a string for debugging purposes and the value there would be `"OBJECT"`:
https://github.com/firebase/firebase-ios-sdk/blob/6059bbbc98a488c096d17c002fae181ab6a548e1/FirebaseVertexAI/Sources/Types/Public/Schema.swift#L60-L63

It's difficult to parse the OpenAPI [spec](https://spec.openapis.org/oas/v3.0.3#data-types) but you can think of an object type like a `Dictionary` in Swift or a JSON object (i.e., key-value pairs).

Reply:
Switched to `DataType.object`.